src/main/java/bio/terra/workspace/common/utils/BaseStatusService.java

```diff
@@ -0,0 +1,81 @@
+package bio.terra.workspace.common.utils;
+
+import bio.terra.workspace.generated.model.SystemStatus;
+import bio.terra.workspace.generated.model.SystemStatusSystems;
```

Comment:
bleh, I know it's generated code, the name `SystemStatusSystems` is gross. Not a blocker for this PR!

thinking out loud ... if we want this to be a basis for other services to extend for their own status endpoints, should it use generated code?

Reply:
Yeah, I'm really not a fan of that name.
We definitely could use this generated code for other services - the `SystemStatus` was one of the original objects in the POC service so other services have the same object. But, that means we're redefining the same thing in a bunch of places, which makes it hard to change later...